osquery/tables/system/posix/python_packages.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#include <string>
+#include <fstream>
+
+#include <boost/algorithm/string/trim.hpp>
+
+#include <osquery/core/conversions.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+
+namespace osquery {
+namespace tables {
+
+const int kNumFields = 2;
+const std::set<std::string> kPythonPath = {
+  "/usr/local/lib/python2.7/", "/usr/lib/python2.7/dist-packages",
+};
+
+
+// checks if given set of paths are sub-path of one another
+int checkOverlap() {
+
+  for (const auto& path : kPythonPath) {
+    int count = 0;
+    for (const auto& key : kPythonPath ) {
+      if (count > 1)
+        return -1;
+      else if (key.find(path) != std::string::npos)
+        count += 1;
+    }
+  }
+
+  return 1;
+}
+
+void genPackage(std::string path, Row &r) {
+  std::ifstream fd (path, std::ios::in | std::ios::binary);
```

Comment:
Hey! I can use that too. I just want to read the file line by line. There's a lot of content after relevant fields so I'm trying to avoid having to read all the content in a string, then parsing it for line breaks and then for fields. Although, I can use `getline` on string too by using `<istringsteam>`. What do you think would be better? I'm open for suggestions :)